app/src/main/java/com/x8bit/bitwarden/data/vault/datasource/disk/VaultDiskSourceImpl.kt

```diff
@@ -124,6 +125,7 @@ class VaultDiskSourceImpl(
     override fun getDomains(userId: String): Flow<SyncResponseJson.Domains> =
         domainsDao
             .getDomains(userId)
+            .filterNotNull()
```

Comment:
So this will be removed when the cherry-pick from my PR goes through, right?

Reply:
Correct